README.md

```diff
@@ -145,3 +146,10 @@ Below is a listing of userspace tools packaged into the node container. The list
 - [`/usr/sbin/runsvchdir`](http://smarden.org/runit/runsvdir.8.html)
     - Starts and monitors a collection of runsv processes
     - Package: `runit`
+
+### License
+
+Calico binaries are licensed under the [Apache v2.0 license](LICENSE), with the exception of some [GPL licensed eBPF code](https://github.com/projectcalico/felix/tree/master/bpf-gpl).
```

Comment:
Suggest changing `GPL licensed eBPF code` to `GPL licensed eBPF programs` to make it clear that they're separate programs.